### PR TITLE
chore: readme instruction improvements

### PR DESCRIPTION
**Pull request by gjermundgaraba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +134,15 @@ just test-foundry
...
 There are several end-to-end tests in the `e2e/interchaintestv8` directory. These tests are written in Go and use the [`interchaintest`](https://github.com/strangelove-ventures/interchaintest) library. It spins up a local Ethereum and a Tendermint network and runs the tests found in [`e2e/interchaintestv8/sp1_ics07_test.go`](e2e/interchaintestv8/sp1_ics07_test.go). Some of the tests use the prover network to generate the proofs, so you need to provide your SP1 network private key to `.env` for these tests to pass.
 
+> If you are running on a mac with an M chip, you will need to do the following:
+> - Set up Rosetta
+> - Enable Rosetta for Docker (in Docker Desktop: Settings -> General -> enable "Use Rosetta for x86_64/amd64 emulation on Apple Silicon")
+> - Pull the foundry image with the following command:
+> 
+>     ```sh
+>     docker pull --platform=linux/amd64 ghcr.io/foundry-rs/foundry:latest
+>     ```
```
Comment (srdtrk) [inline README.md:144]: Shouldn't interchaintest automatically pull it for you?

File: README.md [hunk 2]
```diff
@@ -91,8 +91,15 @@ just build-programs
...
 
-4. Run the Tendermint operator.
+    ```.env
+    CONTRACT_ADDRESS=<CONTRACT_ADDRESS>
+    ```
 
+4. Run the Tendermint operator.
+    
+    To run the operator, you need to select the prover type for SP1. This is set in the `.env` file with the `SP1_PROVER` value (`network|local|mock`).
+    If you run the operator with the `network` prover, you need to provide your SP1 network private key in `.env`.
```
Comment (srdtrk) [inline README.md:101]:
```suggestion
    If you run the operator with the `network` prover, you need to provide your SP1 network private key with `SP1_PRIVATE_KEY=0xyourprivatekey` in `.env`.
```

**Review by srdtrk — APPROVED** [review]
Looks good. Will merge after getting a response to my question

File: README.md (hunk 1 above)
Comment (gjermundgaraba) [reply README.md:144]: For me it pulled the arm version which doesn't work. 🤷

**Pull request merged**